doc/reference/storage_btrfs.md

```diff
@@ -1,6 +1,18 @@
 (storage-btrfs)=
 # Btrfs - `btrfs`
 
+{abbr}`Btrfs (B-tree file system)` is a single-disk file system based on the {abbr}`COW (copy-on-write)` principle.
```

Comment:
Maybe replace `single-disk` with `local` as btrfs actually supports built-in multi-disk layouts including RAID.
Not something we should really cover here but calling it single-disk is inaccurate.